resources/istio-kyma-patch/templates/configmap.yaml

```diff
@@ -77,6 +77,7 @@ data:
   injection-in-namespaces: |
     istio-system disabled
     kube-system disabled
+    kyma-installer disabled
```

Comment:
kyma-installer is our ns so we can label it there directly

Reply:
done